systers/PC-Prep-Kit
# Replaced logos and images with new Peace Corps branding

### Description
Replaced all instances of the old Peace Corps logo with the new logos, including images with the logo edited into it.
Fixes #79 

### Type of Change:
- User Interface

### How Has This Been Tested?
- Check logo used on localhost:3000 page
<img width="1280" alt="screen shot 2018-04-07 at 1 49 25 pm" src="https://user-images.githubusercontent.com/17623615/38458317-0397891a-3a6b-11e8-9152-152d20225977.png">
- Check logo used on Introduction module
<img width="565" alt="screen shot 2018-04-07 at 1 48 57 pm" src="https://user-images.githubusercontent.com/17623615/38458318-06c0e6ea-3a6b-11e8-8374-994e6065bdd3.png">


### Checklist:
- [x ] I have performed a self-review of my own code or materials
